<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit {
    border: 2px solid #b3b3b3;      /* border color */
    border-radius: 12px;            /* rounded corners */
    background: white;
    padding: 5px;                   /* optional */
}

QDateEdit::item {
    padding: 6px;
}</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="timeAppointment">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateTimeEdit {
    border: 2px solid #b3b3b3;      /* border color */
    border-radius: 12px;            /* rounded corners */
    background: white;
    padding: 5px;                   /* optional */
}

QDateTimeEdit::item {
    padding: 6px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>280</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #333333;
    border: 2px solid #e0e0e0;
    border-radius: 10px;
    padding: 8px 16px;
    font-size: 14px;
    font-weight: 500;
}

QPushButton:hover {
    background-color: #f5f5f5;
    border-color: #cfcfcf;
}

QPushButton:pressed {
    background-color: #e8e8e8;
    border-color: #bdbdbd;
}

QPushButton:disabled {
    background-color: #f2f2f2;
    color: #aaaaaa;
    border-color: #dcdcdc;
}



</string>
    </property>
    <property name="text">
     <string>Add Appointment</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableAppointments">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>851</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #333333;
    border: 2px solid #e0e0e0;
    border-radius: 10px;
    padding: 8px 16px;
    font-size: 14px;
    font-weight: 500;
}

QPushButton:hover {
    background-color: #f5f5f5;
    border-color: #cfcfcf;
}

QPushButton:pressed {
    background-color: #e8e8e8;
    border-color: #bdbdbd;
}

QPushButton:disabled {
    background-color: #f2f2f2;
    color: #aaaaaa;
    border-color: #dcdcdc;
}
</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #333333;
    border: 2px solid #e0e0e0;
    border-radius: 10px;
    padding: 8px 16px;
    font-size: 14px;
    font-weight: 500;
}

QPushButton:hover {
    background-color: #f5f5f5;
    border-color: #cfcfcf;
}

QPushButton:pressed {
    background-color: #e8e8e8;
    border-color: #bdbdbd;
}

QPushButton:disabled {
    background-color: #f2f2f2;
    color: #aaaaaa;
    border-color: #dcdcdc;
}
</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>280</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #ffffff;
    color: #333333;
    border: 2px solid #e0e0e0;
    border-radius: 10px;
    padding: 8px 16px;
    font-size: 14px;
    font-weight: 500;
}

QPushButton:hover {
    background-color: #f5f5f5;
    border-color: #cfcfcf;
}

QPushButton:pressed {
    background-color: #e8e8e8;
    border-color: #bdbdbd;
}

QPushButton:disabled {
    background-color: #f2f2f2;
    color: #aaaaaa;
    border-color: #dcdcdc;
}
</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCarModel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCustomerName">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>5</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Unit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Type of maintenance:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="cmbService">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>281</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListWidget {
    border: 2px solid #b3b3b3;      /* border color */
    border-radius: 12px;            /* rounded corners */
    background: white;
    padding: 5px;                   /* optional */
}

QListWidget::item {
    padding: 6px;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtContact">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPlate">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Contact No:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Plate Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmail">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>5</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtOtherMaintenance">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>944</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
